qiskit/ignis/verification/entanglement/analysis.py

```diff
@@ -162,7 +196,7 @@ def get_fourier_info(self, qn, x, y, y_m, p_dict):
         return None
 
 
-def rho_to_fidelity(rho):
+def rho_to_fidelity(rho: float) -> float:
     """
     Get fidelity given rho
```

Comment:
There's no `Args` sections for these. You've added types but no description of how they're used

Reply:
Done, thanks.